changelog/db.changelog-1.1.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
-  <changeSet author="Paul Popa" id="2">
+  <changeSet author="Paul Popa" id="1">
```

Comment:
Although the changes to this file are correct, you can't make changes to a migration that's already been deployed to a higher environment.

Before running migration liquibase generates a checksum of each change request and validates it against migrations applied to the database, if there is any difference in a past migration detected it will fail to run.

So we'll just have to live with the sequence starting at `2`.